src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.resolver.ts

```diff
@@ -125,6 +126,30 @@ export class KeyResultCheckMarkGraphQLResolver extends GuardedNodeGraphQLResolve
     return updatedCheckMark
   }
 
+  @GuardedMutation(KeyResultCheckMarkGraphQLNode, 'key-result-check-mark:update', {
+    name: 'updateCheckMarkAssignee',
+  })
+  protected async updateCheckMarkAssignee(
+    @Args() request: KeyResultCheckMarkUpdateAssigneeRequest,
+    @RequestState() state: GraphQLRequestState,
+  ) {
+    this.logger.log({
+      state,
+      request,
+      message: 'Received update check mark assignee request',
+    })
+
+    const updatedCheckMark = await this.corePorts.dispatchCommand<KeyResultCheckMark>(
+      'update-check-mark-assignee',
+      { id: request.id, ...request.data },
+    )
+
+    if (!updatedCheckMark)
+      throw new UserInputError('We were not able to update this check mark assignee')
+
+    return updatedCheckMark
+  }
+
   @GuardedMutation(DeleteResultGraphQLObject, 'key-result-check-mark:delete', {
     name: 'deleteCheckMark',
   })
```

Comment:
You need to create a `@ResolveField` method, like this: https://github.com/budproj/business/blob/main/src/interface/graphql/modules/key-result/key-result.resolver.ts#L179-L197